source/faq.txt

```diff
@@ -113,33 +113,31 @@ is on the other side of the planet and has a latency of 10000
 milliseconds, setting the ``connectTimeoutMS`` to anything lower will
 prevent the driver from ever connecting to that member.
 
-Should I Use "socketTimeoutMS" as a Way of Preventing Long-Running Operations From Slowing Down the Server?
--------------------------------------------------------------------------------------------------------------
+What Happens to My Operations if My Client Disconnects?
+-------------------------------------------------------
 
-No, you should **not** use ``socketTimeoutMS`` to end operations that
-may run for too long and slow down the application. Attempting to do so
-may not achieve the intended result.
+Starting in {+mdb-server+} version 4.2, the server terminates
+:manual:`certain operations </release-notes/4.2/#client-disconnection>` such as
+aggregations and find operations if the client disconnects before it
+successfully executes an operation.
 
-Closing the socket causes a reconnect of the driver's connection pool,
-introducing latency to any other queued up operations. Chronically slow
-operations will, therefore, cause a large number of reconnect requests,
-negatively impacting throughput and performance.
+Some operations, including all write operations, continue to run on the
+{+mdb-server+} even if the client disconnects. This behavior can cause data
+inconsistencies if your application retries the operation after failure.
 
-Also, closing the socket does not terminate the operation; it will
-continue to run on the MongoDB server, which could cause data
-inconsistencies if the application retries the operation on failure.
+How Can I Confirm That the Driver Closed Unusable Sockets?
+----------------------------------------------------------
 
-However, there are important use cases for ``socketTimeoutMS`` -
-consider the following cases:
+If you experience unexpected network behavior or receive errors when
+executing a MongoDB process, you may not receive confirmation that the
+driver closed the corresponding socket correctly.
```

Comment:
Suggestion:
The adverb "correctly" modifies the closure, so it's usually easier to associate when placed next to word it's related to.

```suggestion
driver correctly closed the corresponding socket.
```